### PR TITLE
Add bigdl backend for BrainMRI example 

### PR DESCRIPTION
**Pull request by ForJadeForest**
## Description
Add the `bigdl` backend for brainMRI example

### CODE REVIEW

File: python/orca/example/learn/pytorch/brainMRI/brainMRI.py [hunk 1]
```diff
@@ -193,11 +193,27 @@ def denormalize(images):
...
 # plot some random training images.
 # You should use jupyter notebook to show the images.
 show_batch(train_loader)
+if args.backend == "bigdl":
+    net = model_creator(config={})
+    optimizer = optim_creator(model=net, config={"lr": 0.001})
+    orca_estimator = Estimator.from_torch(model=net,
+                                          optimizer=optimizer,
+                                          loss=bce_dice_loss,
+                                          metrics=[],
```
Comment (hkvision) [inline python/orca/example/learn/pytorch/brainMRI/brainMRI.py:206]: remove this? metrics will default to be None?

File: python/orca/example/learn/pytorch/brainMRI/brainMRI.py [hunk 2]
```diff
@@ -193,11 +193,27 @@ def denormalize(images):
 }
 train_loader = train_loader_creator(config=config, batch_size=batch_size)
 
+
 # plot some random training images.
 # You should use jupyter notebook to show the images.
 show_batch(train_loader)
+if args.backend == "bigdl":
+    net = model_creator(config={})
```
Comment (hkvision) [inline python/orca/example/learn/pytorch/brainMRI/brainMRI.py:201]: give config a default value None and just call model_creator()?

**Review by hkvision — APPROVED** [review]

**Comment by ForJadeForest** [comment]
Now the customized metrics when use `bigdl` backend are not supported.

**Comment by ForJadeForest** [comment]
[Jenkins test](http://10.112.231.51:18888/job/ZOO-PR-BigDL-Python-ExampleTests-py37-ray/930/)

**Pull request merged**